cql2pgjson/src/test/resources/compoundIndex.sql

```diff
@@ -18,7 +18,20 @@ $func$  LANGUAGE sql IMMUTABLE;
 create or replace function concat_space_sql(VARIADIC text[])
 RETURNS text AS $$ select concat_ws(' ', VARIADIC $1); $$ LANGUAGE SQL IMMUTABLE;
 
+create or replace function concat_array_object_values(jsonb_data text, field text) RETURNS text AS $$
+DECLARE
 
+  res text;
+  i jsonb;
+BEGIN
+  
+  FOR i IN SELECT * FROM json_array_elements(($1)::json)
+   LOOP
+    res =  concat_ws(', ',res, i->$2);
```

Comment:
Can you use ' ' instead of ', '? We don't need commas in the full text matching string.